src/components/SearchPage.js

```diff
@@ -98,7 +98,8 @@ class SearchPage extends React.Component {
 
         // for testing, change cmlgbackend.wdcc to cmlgdevbackend.wdcc
         let url = 'https://cmlgbackend.wdcc.co.nz/api/translations?sequence=' + sequenceTime.getTime() +
-                  '&pageRows=all';
+                  '&pageRows=' + this.state.rowsPerPage;
```

Comment:
why change it here? isn't that we always want to get all the value from backend?